webserver/external/messybrainz.py

```diff
@@ -0,0 +1,34 @@
+import requests
+from flask import current_app
+
+def get_messybrainz_id(querydata):
+    """
+    This function queries messybrainz to get a msid according to the 'artist/title' present in the item metadata
+    Args:
+        querydata: A dictionary containing basic info for one track
+                   minimum fields required: [artis, song title]
+
+    Returns:
+        msid: string messybrainz id -> if it is found in the messybrainz database
+        None: if no reference is fount in messybrainz database
+    """
+    if isinstance(querydata, dict):
```

Comment:
A better way to write this would be to do the opposite check: if it's *not* a dictionary. That way you can more easily follow implementation of that function without having to go deeper and deeper inside `if`s to understand  how it's supposed to work in normal case.